src/test/java/com/github/anba/es6draft/test262/Test262Info.java

```diff
@@ -400,12 +400,12 @@ public void setEs7id(String es7id) {
             this.es7id = es7id;
         }
 
-        public String getId() {
-            return id;
+        public String getEsid() {
+            return esid;
         }
 
-        public void setId(String id) {
-            this.id = id;
+        public void setId(String esid) {
```

Comment:
> Also, sorry for the noise with the whitespace, caused by default settings on the editor.

That's not a problem, the extra whitespace is only present because of the default settings on my editor. Someday I need to fix my settings so Eclipse no longer adds the trailing whitespace in javadoc comments. :smile: